lib/helpers/pils.js

```diff
@@ -0,0 +1,13 @@
+const moment = require('moment');
+
+function attachReviewDue(pil) {
+  return {
+    ...pil,
+    reviewDue: moment(pil.reviewDate).isBefore(moment().add(3, 'months')),
```

Comment:
It feels to me like in the list view this should also be two months. Otherwise the count on the dashboard and the number of due indicators in the list won't align with one another.

Perhaps something like:

```
const attachReviewDue = (months = 3) => pil => {
  return {
    ...pil,
    reviewDue: moment(pil.reviewDate).isBefore(moment().add(months, 'months')),
    ...etc
  };
}